gsa/src/web/pages/reports/alertactions.js

```diff
@@ -81,49 +92,95 @@ class AlertActions extends React.Component {
     }
   }
 
-  handleTestAlert() {
-    const {alert_id} = this.state;
+  handleTriggerAlert(state) {
     const {
+      alertId,
+      applyOverrides,
+      includeNotes,
+      includeOverrides,
+      storeAsDefault,
+    } = state;
+    const {
+      filter,
       gmp,
       report,
-      filter,
       showErrorMessage,
       showSuccessMessage,
     } = this.props;
 
+    filter.set('notes', includeNotes);
```

Comment:
Because set mutates the filter it would be better to create a copy before updating it.
```suggestion
    const newfilter = filter.copy()
    newfilter.set('notes', includeNotes);
```